src/ensembl/package.json

```diff
@@ -13,8 +13,9 @@
   "author": "EMBL-EBI",
   "license": "MIT",
   "scripts": {
-    "start": "npm run serve:dev",
-    "serve:dev": "webpack-dev-server --config ./webpack/webpack.config.dev.js",
+    "copy-dotenv": "test ! -f .env && (cp .env.example .env; echo '.env file created') || true",
+    "start": "npm install --no-save && npm run serve:dev",
```

Comment:
Using `npm install` will slightly (because the packages will be checked against the cache) increase the startup time, but will ensure that the developer always has proper packages.
The --no-save option makes sure that `npm install` does not modify `package-lock.json` (which we'll need to return to the repo, hopefully soon).